### PR TITLE
Adding exit code

### PR DESCRIPTION
**Pull request by sibela46**

### CODE REVIEW

File: cypress/integration/vote_for_card_spec.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ describe('Vote for a card', () => {
...
-    describe('on vote for a card', () => {
-        beforeEach(() => {
-            cy.voteCard();
-        });
-        // it("displays the votes", () => {
-        //     cy.get('[data-cy="vote-card"]').should('not.exist');
-        //     cy.get('[data-cy="votes"]').should('exist');
-        // });
-    });
+    // describe('on vote for a card', () => {
```
Comment (Ro4052) [inline cypress/integration/vote_for_card_spec.js:32]: I would prefer to delete rather than comment.

File: run-cypress-test.js [hunk 2]
```diff
@@ -3,5 +3,13 @@ const server = require('./server/server');
 
 cypress.run()
 .then((results) => {
-  return server.stop()
-});
+  console.log(results.totalFailed);
+  if (results.totalFailed > 0) {
+    return server.fail();
+  }
+  return server.stop();
```
Comment (Ro4052) [inline run-cypress-test.js:10]: Could this then be a one-liner?

`results.totalFailed > 0 ? server.fail() : server.stop()`

That would get rid of the log though.

**Review by Ro4052 — COMMENTED** [review]
You might want to squash these commits 🙈

File: run-cypress-test.js (hunk 2 above)
Comment (sibela46) [reply run-cypress-test.js:10]: I wanted the log for debugging, I can probably change this now

**Pull request closed without merging**